java/src/main/native/src/TableJni.cpp

```diff
@@ -16,9 +16,10 @@
 
 #include <cudf/groupby.hpp>
 #include <cudf/hashing.hpp>
+#include <cudf/partitioning.hpp>
```

Comment:
Duplicate include

Reply:
Do you not need the functionality of `cudf::hash` in `hashing.hpp`?

Reply:
Never mind. I see.